spark-bigquery-dsv2/spark-3.5-bigquery/src/test/java/com/google/cloud/spark/bigquery/integration/Spark35ReadIntegrationTest.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.spark.bigquery.integration;
+
+import static com.google.cloud.spark.bigquery.integration.TestConstants.ALL_TYPES_TABLE_SCHEMA;
+
+import org.apache.spark.sql.types.DataType;
+import org.apache.spark.sql.types.DataTypes;
+import org.apache.spark.sql.types.StructField;
+import org.apache.spark.sql.types.StructType;
+import org.junit.BeforeClass;
+
+public class Spark35ReadIntegrationTest extends ReadIntegrationTestBase {
+  private static StructType allTypesSchemaWithTimestampNTZ;
+
+  @BeforeClass
+  public static void intializeSchema() {
+    allTypesSchemaWithTimestampNTZ = new StructType();
```

Comment:
Since this is copied from the Spark 3.4 test, can you either (1) refactor the original method to be static and call it from here; or (2) inherit from Spark35IntegrationTest  ? Not sure if option 2 will work